src/Lounge.Web/wwwroot/js/stats.js

```diff
@@ -640,6 +640,11 @@ function updateMogiFormatChart(data) {
   );
   const noSQMogiTotal = mogiFormatData.reduce((a, b) => a + b[1], 0);
 
+  const windowSize = Math.max(
```

Comment:
I'm not sure if this is a good approach, it's not dynamic to handle resizing window sizes. I think we should look for a more CSS-based approach rather than a JS one.